libs/components/phone-field/src/lib/modules/phone-field/phone-field.component.spec.ts

```diff
@@ -558,7 +558,7 @@ describe('Phone Field Component', () => {
           '667-555-530',
           '667-555-530',
           false,
-          true,
+          false,
```

Comment:
👇 All of these `touched` checks were wrong previously and addressed by the changes in the pull request.